src/components/map/map.tsx

```diff
@@ -0,0 +1,234 @@
+import React, { useRef, useEffect, useState, useCallback } from 'react';
+import mapboxgl from 'mapbox-gl';
+import style from './map.module.css';
+import 'mapbox-gl/dist/mapbox-gl.css';
+import { mapboxData } from '@atb/modules/org-data';
+import { isFeaturePoint, isStopPlace } from './utils';
+import { ColorIcon } from '@atb/assets/color-icon';
+import Button from '../button/button';
+import { MonoIcon } from '@atb/assets/mono-icon';
+import { ComponentText, useTranslation } from '@atb/translations';
+
+export type LngLatPosition = [number, number];
```

Comment:
I think this should be a record with named parameters. Or at the very least named tuples.
```suggestion
export type LngLatPosition = [lng: number, lat: number]
```